src/scripts/components/structures/Main.js

```diff
@@ -8,11 +8,7 @@ type Props = {
 const MainWrapper = styled.main`
   background-color: ${props => props.theme.colors.body.background};
   flex: 1 1 100%;
-  display: flex;
   padding: 80px;
-  overflow-x: hidden;
-  height: 100vh;
-  justify-content: center;
```

Comment:
Why were these removed? I did see lots of issues before adding this (like you could scroll strangely in the x direction and the page height was odd. Does the app perform correctly at all viewport dimensions now?